docker/src/lemur.conf.py

```diff
@@ -32,7 +32,7 @@ def get_random_secret(length):
                                          base64.b64encode(get_random_secret(32).encode('utf8'))))
 # This must match the key for whichever DB the container is using - this could be a dump of dev or test, or a unique key
 LEMUR_ENCRYPTION_KEYS = repr(os.environ.get('LEMUR_ENCRYPTION_KEYS',
-                                            base64.b64encode(get_random_secret(32).encode('utf8')).decode('utf8')))
+                                            base64.b64encode(b'SBRHNTIW@%#$)_#+ulahitit47774808').decode('utf8')))
```

Comment:
This fixes a bug we have locally where the `LEMUR_ENCRYPTION_KEYS` gets rotated and suddenly queries to the DB no longer work because of a key mismatch. This lemur.conf file is only used for local testing.